src/org/usfirst/frc/team2335/robot/commands/IntakeBalls.java

```diff
@@ -0,0 +1,42 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class IntakeBalls extends Command
+{
+    public IntakeBalls()
+    {
+        requires(Robot.shooter);
+    }
+
+    protected void initialize() 
+    {
+    	
+    }
+
+    protected void execute()
+    {
+    	Robot.shooter.startIntake();
+    }
+
+    // Make this return true when this Command no longer needs to run execute()
```

Comment:
That's an automatically generated comment, it tells you how to use the isFinished() method, this isn't a TODO